src/scripts/check-for-new-lunie-versions.js

```diff
@@ -0,0 +1,36 @@
+/*
+ * this script polls the Lunie site every minute and extracts its etag to check if there has been
+ * any new Lunie versions deployed on Netlify while the user had the app open on the browser
+ *
+ * Usage:
+ *
+ * node tasks/poll-index.js
+ */
+
+const fetch = require("node-fetch")
+
+const checkForNewLunieVersions = () => {
+  let currentEtag = ""
+  let loaded = false
+
+  setInterval(() => {
+    fetch("https://app.lunie.io/index.html").then(result => {
+      console.log("Current Etag", result.headers._headers.etag[0])
+      if (!loaded) {
+        currentEtag = result.headers._headers.etag[0]
+        loaded = true
+      }
+      console.log("Client Etag", currentEtag)
+      if (currentEtag !== result.headers._headers.etag[0]) {
+        console.log("Outdated Lunie version detected! Refreshing...")
+        // the client has an outdated version. We need to refresh the browser to get the latest changes (and chunks)
+        // when the reload method receives true as an argument, it will always reload the page from the server
+        window.location.reload(true)
+        loaded = false
```

Comment:
```suggestion
```

Reply:
Then it won't update the client etag

Reply:
why? `window.location.reload(true)` will reload the whole page

Reply:
oh, and then it reruns the whole function making this unnecessary you mean? 🤔 

Makes sense...